src/morphology_workflows/repair.py

```diff
@@ -422,15 +454,25 @@ def make_release(df, _, zero_diameter_path, unravel_path, repair_path, extension
             df.loc[df["is_valid"], f"zero_diameter_morph_db_path_{extension[:1]}"] = (
                 _zero_diameter_path / "neurondb.xml"
             )
+            df_tmp.loc[df_tmp["is_valid"], f"zero_diameter_morph_db_path_{extension[:1]}"] = (
+                _zero_diameter_path / "neurondb.xml"
+            )
```

Comment:
I think this would be easier to read:
```suggestion
            col_name = f"zero_diameter_morph_db_path_{extension[:1]}"
            db_path = _zero_diameter_path / "neurondb.xml"
            df.loc[df["is_valid"], col_name] = db_path
            df_tmp.loc[df_tmp["is_valid"], col_name] = db_path
```
(and same for the following ones)

Reply:
j'en ai marre de cette PR!

Reply:
:joy: